README.md

```diff
@@ -69,17 +69,17 @@ However, we have created an official core team (core commiters) in order to:
 * Extend the field of technologies known by the review team.
 * Allow several technical opinions on a Proposal/PR, all exchanges are public because we use the GitHub comment feature.
 
-Decision of the core team have the same weight than the projet leaders, so, if a core team member reject a PR (*rejection must be technically documented and explained*) then project leaders will apply the global decision.
+Decision of the core team has the same weight than the projet leaders, so, if a core team member rejects a PR (*rejection must be technically documented and explained*) then project leaders will apply the global decision.
```

Comment:
This whole sentence is actually pretty bad (grammar is wrong, and `project` is spelt wrong). It should either be:
`Decisions of the core team have the same weight to those of the project leaders, so if a core team member rejects...`
or
`A decision from a member of the core team has the same weight to those of the project leaders, so if a core team member rejects.....`

Would you mind updating it to one of those (or something similar that reads well)?

Reply:
Done :)